include/zephyr/device.h

```diff
@@ -197,7 +207,8 @@ typedef int16_t device_handle_t;
 			DEVICE_DT_NAME(node_id), init_fn, pm, data, config,    \
 			level, prio, api,                                      \
 			&Z_DEVICE_STATE_NAME(Z_DEVICE_DT_DEV_ID(node_id)),     \
-			__VA_ARGS__)
+			__VA_ARGS__);                                          \
```

Comment:
shouldn't trailing `;` be conditioned to the next statement? if not, move export above.